src/pdf_beamtime/config/joint_poses.yaml

```
@@ -1,6 +1,11 @@
 pdf_beamtime_server:
   ros__parameters:
-    home_angles: [-0.11536, -1.783732, 0.38816, -1.75492, 0.11484, 3.14159] #For real robot
+    home_angles: [0.0, -1.5708, 0.00, -1.5708, 0.0, 3.14159] #For real robot
     pickup_approach_angles: [4.089481, -0.987856, 2.167873, -1.174083, 0.899019, 3.141593]
     gripper_present: true
     joint_acc_vel_scaling_factor: 0.1
+    joint_constraints:
+      joint_name: 'wrist_3_joint'
+      joint_position: 3.14159
+      upper_limit: 0.78539
+      lower_limit: 0.78539
```

The UR3e has an infinite wrist capability, we should double check that that is turned off in Installation->Safety->Joint Limits on the teach pendant. 

The wrist position should be bound by -2pi -- 2pi in the safety (with some epsilon, they use 363 degrees).  That is, starting at 0, you should be able to complete one full rotation in either direction. I take it that, stepping beyond this rotation resets the value? 

For the movit planning, we can't this be accomplished with  starting at pi, and limiting by pi in either direction? And still maintaining a full range of motion?
```suggestion
      joint_position: 3.14159
      upper_limit: 3.16
      lower_limit: 3.16
```